src/Agent.Sdk/Util/ProcessUtil.cs

```diff
@@ -1,162 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-using System;
-using System.Collections.Generic;
-using System.ComponentModel;
-using System.Diagnostics;
-using System.Linq;
-using System.Management;
-using System.Runtime.InteropServices;
-
-namespace Microsoft.VisualStudio.Services.Agent.Util
```

Comment:
[not-blocking] With the refactoring, it's difficult to review the changes to ProcessUtil; I suggest we hold off on refactoring ProcessUtil for a separate PR, because it doesn't affect the main code path used for this change.